### PR TITLE
Radio checkbox tutorial using Fury API

### PR DESCRIPTION
**Pull request by Nibba2018**
This PR is the continuation of #208 , implemented using FURY API. The following two tutorials are added:

* CheckBox

* Radio Button

The colors of the cube in the CheckBox tutorial does not update due to #280 .

### CODE REVIEW

**Comment by skoudoro** [comment]
Thank you for the update @Nibba2018. I think, we should update `colors_from_actors` before merging this PR. Otherwise, it looks nice.

**Comment by Nibba2018** [comment]
Hey @skoudoro , I have made the required changes.

File: docs/tutorials/02_ui/viz_check_boxes.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+"""
+
+from fury import actor, ui, window, utils
+import numpy as np
+
+###############################################################################
+# We create the corresponding object actors for cube, sphere, cone and arrow.
+
+cube = actor.cube(centers=np.array([[15, 0, 0]]),
+                  colors=np.array([[0, 0, 255]]),
```
Comment (skoudoro) [inline docs/tutorials/02_ui/viz_check_boxes.py:20]: Thank you for the update! I am still waiting for #283 to be merged. Then you can change this from `255` to `1`.

Comment (Nibba2018) [reply docs/tutorials/02_ui/viz_check_boxes.py:20]: Sure 👍

Comment (skoudoro) [reply docs/tutorials/02_ui/viz_check_boxes.py:20]: you can go ahead, it has been merged 😄

Comment (Nibba2018) [reply docs/tutorials/02_ui/viz_check_boxes.py]: @skoudoro Updated 😄

**Comment by skoudoro** [comment]
Thank you @PaulNicolasHunter for creating this tutorial and Thank you @Nibba2018 for fixing this tutorial. merging

closes #208

**Pull request merged**